cmd/gen-go-sample/main.go

```diff
@@ -420,3 +412,13 @@ func (g *generator) handleOut(meth *descriptor.MethodDescriptorProto, valSet Sam
 	}
 	return nil
 }
+
+func prependLines(b *bytes.Buffer, prefix string) {
+	rep := bytes.Replace(b.Bytes(), []byte{'\n'}, []byte("\n"+prefix), -1)
+	b.Reset()
+	b.WriteString(prefix)
+	b.Write(rep)
+
+	// We added the prefix to all newlines, including the ending one, so we overwrote the prefix once.
```

Comment:
I think you mean `s/we overwrite/we'll exclude it/`

Reply:
Rewrote the function.